packages/preview/touying-unistra-pristine/1.0.0/template/template.typ

```diff
@@ -0,0 +1,28 @@
+#import "@preview/touying:0.5.2": *
+#import "@preview/touying-unistra-pristine:1.0.0": *
+
+#show: unistra-theme.with(
```

Comment:
This fails with `unknown variable: unistra-theme`. The reason is that the way you've written `lib.typ` with `#import "unistra.typ" as unistra`, it is actually `unistra.unistra-theme`. You probably want to import the specific items you want to be public in `lib.typ` instead.

Reply:
Indeed — I had the changes in another local folder but forgot to propagate them to the fork. Sorry for the trouble!